controller/JobController.ts

```diff
@@ -14,10 +14,31 @@ module.exports = function(app: Application){
 
     try {
       jobs = await jobservice.getAllJobs();
-      res.render('view-all-jobs', {jobs: jobs, token: req.session.token});
+      res.render('view-all-jobs', {
+        jobs: jobs, 
+        token: req.session.token});
     } catch (error) {
       console.error(error);
     }
     
   });
+
+  app.get('/job-spec/:jobID', roleAccess([UserRole.Admin,UserRole.User]), async (req: Request, res: Response) => {
+    const jobID = parseInt(req.params.jobID);
+
+    try {
+      const job = await jobservice.getJobResponsibilities(jobID); 
+      const responsibilities = await jobservice.getJobResponsibilities(jobID);
```

Comment:
these both call the same thing, you shouldn't need to have 2 calls here

Reply:
removed line 30